### DISCUSSION TITLE
Has anyone tested the oldest OSx version this will work on?

### DISCUSSION THREAD

**Discussion by 3h50** [MartinPham/FindMySync#8] [Q&A]
Just wondering what the oldest smallest OSx version might be that would fully support this :)

**Comment by SimonBard** [comment]
What is your main aim? 

Have a virtual system that takes few ressources?

**Reply by RomanKlein777** [reply]
virtualbox on windows, 3,5GB RAM 2CPUs GB vdi disk.  work nice just for findmysyn. big sur 11.6

**Comment by MartinPham** [comment]
I'm using it on BigSur (via docker-osx)
